packages/designmanual/.storybook/main.js

```diff
@@ -46,6 +46,11 @@ module.exports = {
         'sass-loader',
       ],
     });
+    config.module.rules.push({
+      test: /\.mjs$/,
+      include: /node_modules/,
+      type: 'javascript/auto',
+    });
```

Comment:
Dette er ikke en særlig pen hack, men tror den kan være nødvendig. Vi blander webpack 4 og 5-loadere her. Er overrasket over at det funker, men i og med at det ikke påvirker den faktiske byggeprosessen av pakkene er jeg ikke vond å be.